src/debian/changelog

```diff
@@ -1,3 +1,18 @@
+python-fingerprint (1.6) UNRELEASED; urgency=medium
+
+  * Fixed "port could not be found" on Windows
+  * Fixed image issues with downloadImage()
```

Comment:
Is Markdown supported here? If so, we should write technical details like downloadImage(), charBufferNumber, FINGERPRINT_CHARBUFFER1 etc. in inline code braces

Reply:
Poorly not